src/xenia/config.cc

```diff
@@ -25,12 +25,21 @@ std::shared_ptr<cpptoml::table> ParseFile(
                                    " could not be opened for parsing");
   }
   // since cpptoml can't parse files with a UTF-8 BOM we need to skip them
-  char bom[3];
+  char bom[3]{};
   file.read(bom, sizeof(bom));
   if (file.fail() || bom[0] != '\xEF' || bom[1] != '\xBB' || bom[2] != '\xBF') {
     file.clear();
     file.seekg(0);
   }
+  if (xe::endsWith(filename.string(), ".config.toml")) {
```

Comment:
Did you try just checking if it was null or whatever?
```c++
  if (filename) {
```

Reply:
If `.size()` in `filename` is 0, it won't pass this condition anyway

Reply:
I mean can you check if it's null instead of doing all this?

Reply:
It only freezes when it tries to list the contents of patch files. I added this check to only print stuff from the config